core-bundle/contao/modules/ModuleLogin.php

```diff
@@ -161,7 +161,12 @@ protected function compile()
 			$this->Template->hasError = true;
 			$this->Template->message = $GLOBALS['TL_LANG']['ERR']['invalidTwoFactor'];
 		}
-		elseif ($exception instanceof AuthenticationException)
+		elseif ($isRememberMe && $exception instanceof InsufficientAuthenticationException)
+		{
+			$this->Template->hasError = true;
+			$this->Template->message = $GLOBALS['TL_LANG']['ERR']['insufficientAuthentication'];
+		}
+		elseif ($exception instanceof AuthenticationException && !$exception instanceof InsufficientAuthenticationException)
```

Comment:
And what if `!$isRememberMe && $exception instanceof InsufficientAuthenticationException`?

Reply:
Then we do not want to show a message. The `InsufficientAuthenticationException` is added by our own authenticator, only for the case to detect that re-login was requested by the security authenticator.